tools/distribution/build-xcframework.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+OUTPUT="build"
+PLATFORM="iOS,tvOS"
+
+while :; do
+    case $1 in
+        -o|--output) OUTPUT_FILE=$2
+        shift
+        ;;
+        -p|--platform) PLATFORM=$2
+        shift
+        ;;
+        *) break
+    esac
+    shift
+done
```

Comment:
What does it do 🤔💭😄?

Reply:
That's for the optional arguments :) I've added a `usage` section:
```
./build-xcframework.sh -h
```